driver/controller.go

```diff
@@ -0,0 +1,134 @@
+package driver
+
+import (
+	"errors"
+	"fmt"
+
+	"github.com/Juniper/contrail-go-api"
+	"github.com/Juniper/contrail-go-api/types"
+)
+
+const (
+	DomainName = "default-domain"
```

Comment:
this has to do with multi-region but I'm not entirely sure. In any case, we don't need to deal with it right now.